packages/icons-handlebars/index.js

```diff
@@ -15,15 +15,15 @@ const {
 } = require('@carbon/icon-helpers');
 const { SafeString } = require('handlebars');
 
-function iconHelper(name, { hash = {} } = {}) {
+function iconHelper(name, attributes = {}) {
   const icon = CarbonIcons[name];
   if (!icon) {
     throw new Error(`Unable to find the icon: \`${name}\``);
   }
   const content = icon.content.map(toString);
   const attrs = {
     ...icon.attrs,
-    ...hash,
+    ...(attributes.hash || attributes),
```

Comment:
which would change this to requiring a check for `options.hash` and Object.assign/mergeWith that with `attributes`